doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1690,6 +1696,8 @@ Response code:
           section for details.
           + values - when used, the response payload uses `values` simplified entity representation. See corresponding
           section for details.
+          + unique - when used, the response payload uses `values` simplified entity representation. And, recurring values are left out.
+          See corresponding section for details.
```

Comment:
I have just realized that the missing operation is `GET /v2/types`, in which case unique/distinct doesn't make sense (as entity types returned by that operation are different by definition). Thus, everything seems to be ok (except the typo mentioned in other comment).

NTC